constants.go

```diff
@@ -134,6 +137,8 @@ func VendorFromString(s string) string {
 		return VendorMellanox
 	case strings.Contains(s, "infineon"):
 		return VendorInfineon
+	case strings.Contains(s, "1b4b-9230"):
```

Comment:
It would be worth defining this Marvel PCI ID as a constant and referencing it here,
took me a bit to figure that was a PCI ID :)

Reply:
I agree and made some changes that hopefully clear this up.